src/composables/transactionTokens.ts

```diff
@@ -64,16 +64,6 @@ export function useTransactionTokens({
       ? toShiftedBigNumber(innerTx.value?.fee || 0, -adapter.getAmountPrecision())
```

Comment:
`getAmountPrecision` will fire an error, because it expects and object with `highPrecision` and `amount`.

Reply:
The `BaseProtocolAdapter` has the argument set as not required. But I see your point in the AE adapter as it actually requires the argument. So I updated it in a following way:

Before:
```
  override getAmountPrecision({ highPrecision, amount }: IAmountDecimalPlaces): number {
```

After:
```
  override getAmountPrecision({ highPrecision, amount }: IAmountDecimalPlaces = {}): number {
```

Do you think it solves the problem?